### PR TITLE
Adds Imperium clothing

### PR DESCRIPTION
**Pull request by ondrej008**
Adds code for Imperium clothing, will add icons and stuff later.

:cl: Ondrej008
add: Adds boots, helmets, armour, headset for the Imperial boys.
add: Guardsmen ERT
/:cl:

# FOR THE EMPEROR!

also probably will add an ERT so this *actually* gets used.

- [x] ERT
- [x] Sprites
- [x] Code

### CODE REVIEW

**Comment by Rohesie** [comment]
There is only the uniform for now, according to the iconbot.

**Comment by ondrej008** [comment]
@Rohesie more added

**Comment by Rohesie** [comment]
Alright, let me know when it's ready for a review.

**Comment by ondrej008** [comment]
added

**Comment by ondrej008** [comment]
should they only have their lang or also the normal one?

File: code/datums/emergency_calls/emergency_call.dm [hunk 1]
```diff
@@ -169,9 +169,9 @@
 					if(!istype(M)) //Something went horrifically wrong
 						candidates -= M
 						continue
-					if(M.current?.stat != DEAD)
-						candidates -= M //Strip them from the list, they aren't dead anymore.
-						continue
+					//if(M.current?.stat != DEAD)
```
Comment (DominikPanic) [inline code/datums/emergency_calls/emergency_call.dm]: Why?

Comment (ondrej008) [reply code/datums/emergency_calls/emergency_call.dm]: Although definitely a shitty fix, if you got gibbed or ghosted you wouldn't have the dead flag and you couldn't be a part of the ERT, I can change this back if you want to.

Comment (ondrej008) [reply code/datums/emergency_calls/emergency_call.dm]: I'll have to either make a better check or fix setting flags

File: code/datums/skills.dm [hunk 2]
```diff
@@ -446,4 +446,88 @@
...
 	powerloader = SKILL_POWERLOADER_MASTER
-	large_vehicle = SKILL_LARGE_VEHICLE_TRAINED
\ No newline at end of file
+	large_vehicle = SKILL_LARGE_VEHICLE_TRAINED
+
+//======//I.o.M.\\======\\
+
+/datum/skills/imperial
+	name = "Guardsman"
+	//cqc = SKILL_CQC_DEFAULT
```
Comment (Rohesie) [inline code/datums/skills.dm]: If you want the guardsmen to have zero skills, uncomment this all. Else select their desired skills.

File: code/modules/mob/language.dm [hunk 3]
```diff
@@ -128,6 +128,14 @@
 	colour = "soghun"
 	key = "3"
 
+/datum/language/imperial_english
```
Comment (Rohesie) [inline code/modules/mob/language.dm]: Just name it `imperial`. No need for the English part.

**Comment by ondrej008** [comment]
Don't merge, I made an oopsie and broke the language, I'll fix it tomorrow, also trying out sprites

**Comment by Rohesie** [comment]
Let me know when it's ready for another test.

**Comment by Rohesie** [comment]
Solve conflicts so we can test it again.

**Comment by DominikPanic** [comment]
If you need help adapting this to the new job system do poke me.

**Comment by juanototo2** [comment]
>imperial language
Rename it to Low Gothic please, it tickles my autism

**Comment by Rohesie** [comment]
Code seems ok. Is it tested locally?

File: code/game/jobs/job/imperium.dm [hunk 4]
```diff
@@ -0,0 +1,152 @@
+/datum/job/imperial
```
Comment (DominikPanic) [inline code/game/jobs/job/imperium.dm]: Make it follow the other files organization please:
job
outfit
job
outfit

instead of
job
job
outfit 
outfit

Comment (ondrej008) [reply code/game/jobs/job/imperium.dm]: k

**Comment by ondrej008** [comment]
Yes, it is tested locally.

**Comment by Alterist** [comment]
jumpsuit shouldn't have boot sprites, and armor icon sprites should be consistent across variants

**Comment by Rohesie** [comment]
The ERT currently spawns naked.

**Pull request merged**